config/routes/find.rb

```diff
@@ -1,3 +1,7 @@
+constraints(FindConstraint.new) do
```

Comment:
Do we still need this then?

:thinking:

Reply:
Yup - this is what actually sets the root path to go to the find kickoff page (if coming from the temp url)